sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -1148,34 +1142,16 @@ private UUID saveErrorLogFiles(Throwable throwable, ManagedErrorLog errorLog) th
         String filename = errorLogId.toString();
         AppCenterLog.debug(Crashes.LOG_TAG, "Saving uncaught exception.");
         File errorLogFile = new File(errorStorageDirectory, filename + ErrorLogHelper.ERROR_LOG_FILE_EXTENSION);
+
+        /* Set stacktrace to error log. */
+        if (errorLog.getException() != null) {
```

Comment:
The check is unnecessary since we don't expect the exception to be null here.
It comes either from @NonNull final Exception exception
```
@NonNull
    public static ManagedErrorLog createErrorLog(@NonNull Context context, @NonNull final java.lang.Thread thread, @NonNull final Exception exception, @NonNull final Map<java.lang.Thread, StackTraceElement[]> allStackTraces, final long initializeTimestamp, boolean fatal) {
```
or from
```
Exception modelException = new Exception();
modelException.setType("minidump");
modelException.setWrapperSdkName(WRAPPER_SDK_NAME_NDK);
modelException.setMinidumpFilePath(dest.getPath());
ManagedErrorLog errorLog = new ManagedErrorLog();
```